crates/storage/codecs/derive/src/compact/mod.rs

```diff
@@ -143,7 +143,7 @@ fn should_use_alt_impl(ftype: &String, segment: &syn::PathSegment) -> bool {
                 if let (Some(path), 1) =
                     (arg_path.path.segments.first(), arg_path.path.segments.len())
                 {
-                    if ["B256", "Address", "Address", "Bloom", "TxHash"]
+                    if ["B256", "Address", "Address", "Bloom", "TxHash", "BlockHash"]
```

Comment:
is this breaking/will it change the codec?

Reply:
no, this is actually the first field where use `BlockHash` with `Compact` (I'm sure we use B256 in some fields)